src/main/java/seedu/address/logic/commands/EditClientCommand.java

```diff
@@ -119,8 +127,20 @@ private static Client createEditedClient(Client clientToEdit, EditClientDescript
             : editClientDescriptor.getBirthday();
         Set<Product> updatedProducts = editClientDescriptor.getProducts().orElse(clientToEdit.getProducts());
 
-        return new Client(updatedName, updatedPhone, updatedEmail, updatedAddress,
+        Client client = new Client(updatedName, updatedPhone, updatedEmail, updatedAddress,
                 updatedBirthday, updatedProducts);
+
+        // update client in each meeting this client has
+        List<Meeting> meetings = clientToEdit.getMeetings();
```

Comment:
why is editClient Descriptor not used for meetings? the other parts of the methods all uses editClientDescriptor